### DISCUSSION TITLE
What is the duration of media retention by GitHub for attached files within issues or any other areas that have been uploaded but not posted?

### DISCUSSION THREAD

**Discussion by Yawan-1** [community/community#77006] [Projects and Issues]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Yesterday, I was about to open a issue in which I wanted to add a media but then I discarded the issue but the media wasn't really deleted from the github's assets and It is still showing when I visit to that uploaded image's path. 

So How long does github keep the media/assets which uploaded but never posted to see someone?

### Guidelines

- [x] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by notsointresting** [comment]
GitHub retains uploaded but unlinked media for 30 days. After that, if the media is not linked in any issue, pull request, comment, or other GitHub location, it may be deleted.

**Reply by dt-alexandre-weisser** [reply]
Hi ! 

Thanks for this answer but I couldn't any official source online to match this, could you point me toward some official github documentation or declaration that confirms this information ?

**Reply by Simran-B** [reply]
Only being used in a project's README doesn't seem to suffice, my images went missing after approximately 30 days.

**Reply by ryanorsinger** [reply]
What about the case where the uploaded media (say from a copy/paste of an image) is used in a Wiki that's hosted on GitHub or a README.md file? This would help us build documentation more rapidly. Thanks!
